features/dbPing.js

```diff
@@ -0,0 +1,58 @@
+var mysql = require("mysql2");
+
+module.exports = function (app) {
+  app.message("dbping", dbPing);
+};
+
+async function dbPing({ message, client }) {
+  // Connect to the database
+  var con = mysql.createConnection({
+    host: process.env.MYSQL_HOSTNAME,
+    user: process.env.MYSQL_USER,
+    password: process.env.MYSQL_PASSWORD,
+    database: process.env.MYSQL_DATABASE,
+  });
+
+  con.connect(function (err) {
+    if (err) throw err;
+  });
+
+  // First we will make the table if there isn't one
+  con.query(
+    {
+      sql: "CREATE TABLE IF NOT EXISTS profiles(name varchar(20) PRIMARY KEY, pings INT)",
+      timeout: 30000, // 30s
+    },
+    function (err) {
+      if (err) throw err;
+    }
+  );
+
+  // Then we will INSERT/UPDATE the current ping value
+  con.query(
```

Comment:
because this query and the query above aren't promises, it's possible that these might execute in an order that you don't expect.  it probably won't happen, but we should try to improve the way we use the `mysql2` library to ensure that we're properly treating some of these function calls as asynchronous.